### PR TITLE
connected calendar

### PR DESCRIPTION
**Pull request by jlavendtt**

### CODE REVIEW

**Review by jaitken — APPROVED** [review]
LGTM

File: src/components/ListingShow/ListingShow.vue [hunk 1]
```diff
@@ -19,7 +19,11 @@
             <Amenities  v-if="listing" :id="listing.id"/>
           </div>
           <div class="col-12">
-              <h1>Calendar</h1>
+              <v-app>
```
Comment (Eliotdoesprogramming) [inline src/components/ListingShow/ListingShow.vue:22]: Shouldn't need a separate vue app inside our vue app

File: src/components/Calendar/Calendar.vue [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+  mounted(){
+        axios.get('http://localhost:8080/api/event/getByListingId/'+this.listing.id)
+        .then(response => {
+            this.list = response.data;
+
+            
+            
+        })
+
```
Comment (Eliotdoesprogramming) [inline src/components/Calendar/Calendar.vue:187]: missing the @click methods, creating errors in console.

**Review by Eliotdoesprogramming — CHANGES_REQUESTED** [review]
Tested and you can remove the v-app and  v-content around the calendar. Also remove the @click handlers with undefined methods (causing errors in js will have ripple effects on the whole page)

Noticed that with this PR, the GMap component gets pushed to the same as its title which I thought was odd because I didn't see any changes directly to it, my guess would be that its related to the size of the calendar component affecting rows somehow.

I reviewed Roberto's PR earlier and I think he may have been expecting a calendar to be put in as a dialog for his price component. May be good to coordinate with him.

**Pull request merged**